eos/parameters/polarisations.yaml

```diff
@@ -5,6 +5,13 @@
     max:     +0.655
     comments: "cf. [PDG2012+]"
 
+# Decay parameter of the Lambda_c baryon from FOCUS collaboration
+"Lambda_c::alpha@FOCUS" :
+    central: -0.780
+    min:     -0.986
+    max:     -0.574
+    comments: "cf. [hep-ex/0509042]"
```

Comment:
@akokulu7 I'd prefer if you used the EOS internal referencing. Also, PDG completely suffices here.